packages/sdk/src/types/notification/ConnectorNotificationId.ts

```diff
@@ -0,0 +1,3 @@
+export interface ConnectorNotificationId {
+    notificationId: string;
+}
```

Comment:
the naming of this interface is not good.

It's used to describe a Response that is including  a notification id. Instead you should create one response object for every Method that returns such a response.